environments/icds-cas/public.yml

```diff
@@ -299,73 +299,73 @@ localsettings:
   ICDS_DOMAIN: "{{ localsettings_private.ICDS_DOMAIN }}"
   UCR_COMPARISONS:
     # MPR
-    static-mpr_1_person_cases:
-    static-mpr_2a_person_cases:
-    static-mpr_2a_3_child_delivery_forms:
-    static-mpr_2bi_preg_delivery_death_list:
-    static-mpr_2bii_child_death_list:
-    static-mpr_2ci_child_birth_list:
-    static-mpr_3i_person_cases:
-    static-mpr_3ii_person_cases:
-    static-dashboard_mpr_4a_6_pse:
-    static-mpr_4a_6_pse:
-    static-mpr_4b_infra_forms:
-    static-mpr_5_child_health_cases:
-    static-mpr_5_child_health_cases_v2:
-    static-mpr_5_ccs_record_cases:
-    static-mpr_5_ccs_record_cases_v2:
-    static-mpr_6ac_child_health_cases:
-    static-mpr_6ac_child_health_cases_v2:
-    static-mpr_6b_child_health_cases:
-    static-mpr_6b_child_health_cases_v2:
-    static-mpr_7_growth_monitoring_forms:
-    static-mpr_8_tasks_cases:
-    mpr_9_vhnd_forms:
-    static-mpr_10a_person_cases:
-    static-mpr_10ab_person_cases:
-    static-mpr_10b_person_cases:
-    static-mpr_11_visitor_book_forms:
+    static-icds-cas-static-mpr_1_person_cases:
+    static-icds-cas-static-mpr_2a_person_cases:
+    static-icds-cas-static-mpr_2a_3_child_delivery_forms:
+    static-icds-cas-static-mpr_2bi_preg_delivery_death_list:
+    static-icds-cas-static-mpr_2bii_child_death_list:
+    static-icds-cas-static-mpr_2ci_child_birth_list:
+    static-icds-cas-static-mpr_3i_person_cases:
+    static-icds-cas-static-mpr_3ii_person_cases:
+    static-icds-cas-static-dashboard_mpr_4a_6_pse:
+    static-icds-cas-static-mpr_4a_6_pse:
+    static-icds-cas-static-mpr_4b_infra_forms:
+    static-icds-cas-static-mpr_5_child_health_cases:
+    static-icds-cas-static-mpr_5_child_health_cases_v2:
+    static-icds-cas-static-mpr_5_ccs_record_cases:
+    static-icds-cas-static-mpr_5_ccs_record_cases_v2:
+    static-icds-cas-static-mpr_6ac_child_health_cases:
+    static-icds-cas-static-mpr_6ac_child_health_cases_v2:
+    static-icds-cas-static-mpr_6b_child_health_cases:
+    static-icds-cas-static-mpr_6b_child_health_cases_v2:
+    static-icds-cas-static-mpr_7_growth_monitoring_forms:
+    static-icds-cas-static-mpr_8_tasks_cases:
+    static-icds-cas-mpr_9_vhnd_forms:
```

Comment:
should this have an extra static?

Reply:
for whatever reason this one was named differently:

```
custom/icds_reports/ucr/reports/mpr/mpr_9_vhnd_forms.json
24:  "report_id": "mpr_9_vhnd_forms",
```

```
custom/icds_reports/ucr/reports/mpr/mpr_8_tasks_cases.json
24:  "report_id": "static-mpr_8_tasks_cases",
```